Cargo.toml

```diff
@@ -15,6 +19,7 @@ grin_grin = { path = "./grin" }
 grin_config = { path = "./config" }
 grin_core = { path = "./core" }
 grin_pow = { path = "./pow"}
+grin_p2p = { path = "./p2p"}
 grin_util = { path = "./util"}
 secp256k1zkp = { git = "https://github.com/mimblewimble/rust-secp256k1-zkp" }
```

Comment:
Just noticed this here.. while you're here, this shouldn't be importing secp256k1 directly here, (it should be using the version exported from util) Is this actually being used directly from src/grin.rs?

Reply:
Indeed, removing it cause no problems as it doesn't look like to be used by src/grin.rs

Reply:
Okay, thanks! (I know it's tangental to what you're doing) Would you mind making that change and committing (to avoid future confusion). Thanks!

Reply:
Sure!